### DISCUSSION TITLE
Allow adding custom HTML tag for Elementor Fullwidth Template inside Page Settings

### DISCUSSION THREAD

**Discussion by Henzxon** [elementor/elementor#23982] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm always frustrated when containers and nested containers can be tagged with proper semantic HTML and landmarks but not the Elementor Fullwidth Template

### Describe the solution you'd like

It would be nice if an HTML tag dropdown is also available inside the page settings no only for the containers.

### Describe alternatives you've considered

Creating a single page template and make everything full-width with zero padding all side and add a post content and add an HTML tag as main for the container but this will disable the plus handle when editing elementor pages.

I tried doing this but the client accidentally modified the single post template and broke the site. Never used it again.

The semantic structure is now Header > Div > Footer instead of Header > Main/Aside > Footer

Workaround: https://youtu.be/96NOwnZGpHM

### Additional context

<img width="973" alt="image" src="https://github.com/elementor/elementor/assets/102000469/aa9b12e3-3def-455c-a7eb-a0ea6f68c2ac">

**Comment by carlos-blazquez-global** [comment]
+1
